### ISSUE TITLE
About error of deploying the twilio chatbot

### ISSUE THREAD

**Issue by michael7908** [pipecat-ai/pipecat#251]
Hi There, I came across an error after all deployments were in place and it seems that I need a full Twilio account instead of a trial one.

Is that correct to fully use this chatbot powered by LLMs?

Thanks.

**Comment by aconchillo** [comment]
> Hi There, I came across an error after all deployments were in place and it seems that I need a full Twilio account instead of a trial one.
> 
> Is that correct to fully use this chatbot powered by LLMs?
> 
> Thanks.

Ideally yes. I initially tried with a trial account and I found these issues:

- When I tried calling with my US phone I got a welcome message from Twilio sayings something like "If you want to remove this message upgrade to a full account, press any key to continue". And then after pressing a key on the phone it just worked.

- When I used Google Voice to make a call I believe Twilio was rejecting it because it just didn't work. When I upgraded to a full account it worked.
